ios/FluentUI/People Picker/AvatarStack.swift

```diff
@@ -0,0 +1,160 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+import UIKit
+
+@objc(MSFAvatarStack)
+open class AvatarStack: UIView {
+    /// Array of avatars to display in the avatar stack.
+    @objc open var avatars: [Avatar] {
```

Comment:
i feel like we need to open the door to let clients to have "+" button at the end or show "+10" somehow. it is pretty limited with just avatar array.

Reply:
maybe we need to support overflow style of avatar shown in toolkit figma?

Reply:
![Screen Shot 2020-08-03 at 9 11 15 PM](https://user-images.githubusercontent.com/20715435/89252200-e712c380-d5cd-11ea-866b-aded473b7be1.png)